### DISCUSSION TITLE
Media pipe : multipose support

### DISCUSSION THREAD

**Discussion by 60-hz** [cables-gl/cables_docs#720] [Feature Requests]
In [official documentation of MediaPipe](https://developers.google.com/mediapipe/solutions/vision/pose_landmarker), there is a new "num_poses" configurations options.

`num_poses	The maximum number of poses that can be detected by the Pose Landmarker.	Integer > 0`

Would be great to be able to use it in Cables.gl as well 

More infos:
https://gist.github.com/lanzani/f85175d8fbdafcabb7d480dd1bb769d9

**Comment by mariomatasystems** [comment]
Would also love this!

**Comment by MoorPoor** [comment]
I hope I understood the question correctly and I can just agree that being able to recognize more than one person in pose tracking would be great. It's a bit of a shame when you're working on an installation and only realize at the end that you've been sitting alone in front of the computer the whole time... classic ;)
I wonder if it would help to use just more than one Ops for more than one person to be recognized.
